src/components/Reviews.jsx

```diff
@@ -2,18 +2,21 @@ import { useEffect } from "react"
 import { useState } from "react"
 import { getReviews } from "../api"
 import ReviewCard from "./ReviewCard"
+import { useSearchParams } from "react-router-dom"
```

Comment:
Really like your choice of using search params for the category filter, this can also be done with another route but realy like this way of doing it.